src/pods/edit-realtion/edit-relation.business.ts

```diff
@@ -1,5 +1,10 @@
-import { DropdownOptionVm } from '@/common/components';
-import { DatabaseSchemaVm } from '@/core/providers/canvas-schema';
+import { DropdownOptionVm, PkOptionVm } from '@/common/components';
+import { GUID } from '@/core/model';
+import {
+  DatabaseSchemaVm,
+  FieldVm,
+  TableVm,
+} from '@/core/providers/canvas-schema';
 
 export const mapRelationsTipeToDropdonwVm = (): DropdownOptionVm[] => [
```

Comment:
Mention this issue (unit tests)

https://github.com/Lemoncode/mongo-modeler/issues/118

Reply:
It's fixed